src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -515,4 +558,4 @@ export default compose(
     }),
     withLocalize,
     withPolicy,
-)(ReimbursementAccountPage);
+)(ReimbursementAccountPage);
```

Comment:
Add an empty line.

Reply:
Still a last line is missed.

<img width="352" alt="Screenshot 2023-10-11 at 11 52 56" src="https://github.com/Expensify/App/assets/57314631/20ddd465-f371-4809-88a6-147ef397f393">